### PR TITLE
Create validation data inside the Pulp's working directory

### LINKED ISSUE #2798: Can't create signing service
**Version**
3.20

**Describe the bug**
When attempting to create a singing service using the `pulpcore-manager add-signing-service` command I get the following error:

```
api_1          | Traceback (most recent call last):
api_1          |   File "/venv/bin/django-admin", line 8, in <module>
api_1          |     sys.exit(execute_from_command_line())
api_1          |   File "/venv/lib64/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
api_1          |     utility.execute()
api_1          |   File "/venv/lib64/python3.8/site-packages/django/core/management/__init__.py", line 413, in execute
api_1          |     self.fetch_command(subcommand).run_from_argv(self.argv)
api_1          |   File "/venv/lib64/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
api_1          |     self.execute(*args, **cmd_options)
api_1          |   File "/venv/lib64/python3.8/site-packages/django/core/management/base.py", line 398, in execute
api_1          |     output = self.handle(*args, **options)
api_1          |   File "/venv/lib64/python3.8/site-packages/pulpcore/app/management/commands/add-signing-service.py", line 83, in handle
api_1          |     SigningService.objects.create(
api_1          |   File "/venv/lib64/python3.8/site-packages/django/db/models/manager.py", line 85, in manager_method
api_1          |     return getattr(self.get_queryset(), name)(*args, **kwargs)
api_1          |   File "/venv/lib64/python3.8/site-packages/django/db/models/query.py", line 453, in create
api_1          |     obj.save(force_insert=True, using=self.db)
api_1          |   File "/venv/lib64/python3.8/site-packages/pulpcore/app/models/content.py", line 826, in save
api_1          |     self.validate()
api_1          |   File "/venv/lib64/python3.8/site-packages/pulpcore/app/models/content.py", line 858, in validate
api_1          |     with tempfile.TemporaryDirectory(dir=".") as temp_directory_name:
api_1          |   File "/usr/lib64/python3.8/tempfile.py", line 780, in __init__
api_1          |     self.name = mkdtemp(suffix, prefix, dir)
api_1          |   File "/usr/lib64/python3.8/tempfile.py", line 358, in mkdtemp
api_1          |     _os.mkdir(file, 0o700)
api_1          | PermissionError: [Errno 13] Permission denied: './tmpcmbnyssa'
```

The error seems to be caused by this change https://github.com/pulp/pulpcore/commit/d32c62a2da520e696e30504f6de98c1dfc4406e7#diff-7c39d38f1aefe19ba0c60a9506034c8258a013de8b65c529c7d89c581053f05dR177, which attempts to create a new temporary directory in `.`, which my deployment doesn't have permissions to write to. I have no idea what directory `.` corresponds to here, so I can't make it writable.

**To Reproduce**
- attempt to create a signing service with `pulpcore-manager add-signing-service`

**Expected behavior**
I should be able to added a signing service

### PR DESCRIPTION
**Pull request by lubosmj**
closes #2798

### CODE REVIEW

**Review by newswangerd — APPROVED** [review]

File: CHANGES/2798.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed a bug that disallowed users to create a signing service via the CLI.
```
Comment (ipanova) [inline CHANGES/2798.bugfix:1]: via the pulpcore-manager

Comment (ipanova) [inline CHANGES/2798.bugfix:1]: s/users/administrators/

**Review by ipanova — APPROVED** [review]

**Comment by newswangerd** [comment]
FYI this pattern is used in import export as well. https://github.com/pulp/pulpcore/blob/main/pulpcore/app/importexport.py#L107 ([other instances](https://github.com/pulp/pulpcore/search?q=TemporaryDirectory)).

Not sure if it's intentional, I don't have a way to test it and I don't think it should block this PR, but I wanted to call it out just in case.

**Comment by ipanova** [comment]
> FYI this pattern is used in import export as well. https://github.com/pulp/pulpcore/blob/main/pulpcore/app/importexport.py#L107 ([other instances](https://github.com/pulp/pulpcore/search?q=TemporaryDirectory)).
> 
> Not sure if it's intentional, I don't have a way to test it and I don't think it should block this PR, but I wanted to call it out just in case.

those should be ok since they are called in a task

**Review by bmbouter — APPROVED** [review]

**Pull request merged**